server-mysql/changeDatabase.xsl

```diff
@@ -8,7 +8,7 @@
 
     <xsl:template match="//ds:subsystem/ds:datasources/ds:datasource[@jndi-name='java:jboss/datasources/KeycloakDS']">
         <ds:datasource jndi-name="java:jboss/datasources/KeycloakDS" pool-name="KeycloakDS" enabled="true" use-java-context="true" use-ccm="true">
-            <ds:connection-url>jdbc:mysql://${env.MYSQL_PORT_3306_TCP_ADDR}:${env.MYSQL_PORT_3306_TCP_PORT}/${env.MYSQL_DATABASE:keycloak}</ds:connection-url>
+            <ds:connection-url>jdbc:mysql://${env.MYSQL_HOST}:${env.MYSQL_PORT:3306}/${env.MYSQL_DATABASE:keycloak}</ds:connection-url>
```

Comment:
The above is an example of the env vars auto-generated when using docker linking :( Converting to compose allows us to clean them up :)